File: backend/src/main/java/com/team05/petmeeting/domain/naming/dto/BadWordListRes.java
package com.team05.petmeeting.domain.naming.dto;

import java.util.List;

// 금칙어 전체 조회 응답 DTO
public record BadWordListRes(
        List<BadWordDto> badWords,
        int totalCount
) {
    public record BadWordDto(
            Long wordId,
            String word,
            String addedAt
    ) {}
}

File: backend/src/main/java/com/team05/petmeeting/domain/naming/dto/BadWordListRes.kt
package com.team05.petmeeting.domain.naming.dto

// 금칙어 전체 조회 응답 DTO
data class BadWordListRes(
    val badWords: List<BadWordDto>,
    val totalCount: Int
) {
    @JvmRecord
    data class BadWordDto(
        val wordId: Long,
        val word: String,
        val addedAt: String
    )
}